literature.bib

```diff
@@ -1282,11 +1232,11 @@ @inproceedings{HMS+:SPLC22
 	year = 2022
 }
 
-% formerly known as KTS+2022arxiv
 @techreport{KTS+:TR22,
+	renamedFrom = {KTS+2022arxiv},
```

Comment:
Looks good to me but this key should be documented in the README. The README currently says to use comments as the one removed here.

Reply:
fixed